worldedit-core/src/main/java/com/fastasyncworldedit/core/object/io/AsyncBufferedOutputStream.java

```diff
@@ -1,183 +0,0 @@
-package com.fastasyncworldedit.core.object.io;
-
-import java.io.FilterOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.concurrent.ConcurrentLinkedDeque;
-
-/**
- * BufferedOutputStream that asynchronously flushes to disk, so callers don't
- * have to wait until the flush happens. Buffers are put into a queue that is
- * written asynchronously to disk once it is really available.
- *
- * <p>
- * The error handling (as all stream ops are done asynchronously) is done during
- * write and close. Exceptions on the asynchronous thread will be thrown to the
- * caller either while writing or closing this stream.
- * </p>
- *
- * @apiNote This class is thread-safe.
- * @author thomas.jungblut
- */
-public final class AsyncBufferedOutputStream extends FilterOutputStream {
```

Comment:
This makes me nervous

Reply:
This class was used nowhere.

Reply:
Just the class name and what it does lol